packages/test/test-end-to-end-tests/src/test/gc/gcAttachmentBlobs.spec.ts

```diff
@@ -16,7 +16,7 @@ import { describeNoCompat, ITestDataObject } from "@fluidframework/test-version-
 import { getUrlFromItemId, MockDetachedBlobStorage } from "../mockDetachedBlobStorage";
 import { getGCStateFromSummary } from "./gcTestSummaryUtils";
 
-const ensureContainerConnectedWriteMode = async (container: Container) => {
+const waitForContainerConnectionWriteMode = async (container: Container) => {
```

Comment:
Not part of this PR but I think this should be a utility too. It's used in 2-3 tests.